tools/docker/build_docker_util.py

```
@@ -0,0 +1,124 @@
+    return architecture
+
+def tag_and_push_image(dockerimage: str, tag: str, registry: str) -> None:
+    """Tag and push the image to a registry (e.g. Github or Scaleway).
+
+       Args:
+          dockerimage (str): Name of the Docker image.
+          tag (str): Image tag
+     """
+    registry_image_name = f'{registry}/{dockerimage}:{tag}'
+    print(f"Tagging image as {registry_image_name}")
+    subprocess.run(['docker', 'tag', f'{dockerimage}:{tag}', registry_image_name], check=True)
```

Ruff (0.14.4)

`subprocess` call: check for execution of untrusted input

(S603)